epubmaker_preprocessing.js

```diff
@@ -62,26 +62,53 @@ fs.readFile(file, function editContent (err, contents) {
 
   // add chap numbers to chap titles if specified
   $("h1[data-labeltext]").each(function () {
-      var labeltext = $(this).attr('data-labeltext');
-      if (labeltext.trim()) {
-        $(this).prepend(labeltext + ": "); 
-      };   
+    var labeltext = $(this).attr('data-labeltext');
+    if (labeltext.trim()) {
+      $(this).prepend(labeltext + ": "); 
+    };   
   });
 
-$("span.spanhyperlinkurl:not(:has(a))").each(function () {
-      var newlink = "<a href='" + $( this ).text() + "'>" + $( this ).text() + "</a>";
-      var mypattern1 = new RegExp( "https?://", "g");
-      var result1 = mypattern1.test($( this ).text());
-      var mypattern2 = new RegExp( "^@", "g");
-      var result2 = mypattern2.test($( this ).text());
-      if (result1 === false && result2 === false) {
-        newlink = newlink.replace("href='", "href='http://");
-      }
-      if (result1 === false && result2 === true) {
-        newlink = newlink.replace("href='@", "href='https://twitter.com/");
-      }
-      $(this).empty();
-      $(this).prepend(newlink); 
+  // remove any reference to printing in the copyright page
+  // and insert the correct copyright symbol on copyright page
+  $("section[data-type='copyright-page'] p").each(function () {
+    var myHTML = $( this ).html().replace(/Printed in [a-zA-Z\s]+\./g, '').replace(/([C|c]opyright)(\s|&.*?;)+/g, '$1 &#169; ');
+    $(this).empty();
```

Comment:
Ah, good catch. They're exactly the same, functionally. The extra spaces just make it easier to read. I suppose I should decide on a standard usage.